src/main/java/com/mathworks/ci/RunMatlabCommandBuilder.java

```diff
@@ -0,0 +1,137 @@
+package com.mathworks.ci;
+/**
+ * Copyright 2019-2020 The MathWorks, Inc.
+ * 
+ * Script builder used to run custom MATLAB commands or scripts.
+ * 
+ */
+
+import java.io.IOException;
+import javax.annotation.Nonnull;
+import org.jenkinsci.Symbol;
+import org.kohsuke.stapler.DataBoundConstructor;
+import org.kohsuke.stapler.DataBoundSetter;
+import org.kohsuke.stapler.StaplerRequest;
+import hudson.EnvVars;
+import hudson.Extension;
+import hudson.FilePath;
+import hudson.Launcher;
+import hudson.Launcher.ProcStarter;
+import hudson.model.AbstractProject;
+import hudson.model.Result;
+import hudson.model.Run;
+import hudson.model.TaskListener;
+import hudson.tasks.BuildStepDescriptor;
+import hudson.tasks.Builder;
+import jenkins.tasks.SimpleBuildStep;
+import net.sf.json.JSONObject;
+
+public class RunMatlabCommandBuilder extends Builder implements SimpleBuildStep {
+    private int buildResult;
+    private EnvVars env;
+    private String matlabCommand;
+
+    @DataBoundConstructor
+    public RunMatlabCommandBuilder() {
+
+    }
+
+
+    // Getter and Setters to access local members
+
+
+    @DataBoundSetter
+    public void setMatlabCommand(String matlabCommand) {
+        this.matlabCommand = matlabCommand;
+    }
+
+    public String getMatlabCommand() {
+        return this.matlabCommand;
+    }
+
+    private String getCommand() {
+        return this.env == null ? getMatlabCommand() : this.env.expand(getMatlabCommand());
+    }
+
+    private void setEnv(EnvVars env) {
+        this.env = env;
+    }
+
+    private EnvVars getEnv() {
+        return this.env;
+    }
+
+    @Symbol("RunMatlabCommand")
+    @Extension
+    public static class MatlabScriptDescriptor extends BuildStepDescriptor<Builder> {

```

Comment:
Descriptor class name needs to be updated to match the new Wrapper name.

Reply:
Done